### DISCUSSION TITLE
Is it possible to adjust the spacing between the icons and the filenames?

### DISCUSSION THREAD

**Discussion by ayala** [eza-community/eza#725] [Q&A, Howto, Help] [answered]
I find they are too close for me.

TIA

**Comment by gierens** [comment, marked as answer]
There is an environment variable, `EZA_ICON_SPACING`, for this:
![Screenshot from 2024-01-22 09-51-18](https://github.com/eza-community/eza/assets/49617392/3d81564a-881b-4b5c-9879-00f3089eaeee)
So you can configure it accordingly in your shell's rc file (`.zshrc`, `.bashrc`, ...) or profile.

**Reply by ayala** [reply]
nice!!! thank you!

**Reply by gierens** [reply]
Great, glad I could help! :) ... Can you mark it as answer? Then it shows up as answered in the discussion overview :)
